app/validators/pem_der_key_validator.rb

```diff
@@ -0,0 +1,10 @@
+require "openssl"
+
+# Verifies that an attribute is a valid PEM/DER encoded key
+class PemDerKeyValidator < ActiveModel::EachValidator
```

Comment:
This validator seems to fail, when trying to upload a password protected key.

I think it is not supposed to be password-protected as a mutual TLS key, but maybe we should put this in the UI.